analysis_file/analysis_file.py

```diff
@@ -78,13 +81,23 @@
     data = FoldData.fold(user, data, group_by_fn, equal_keys, concat_keys, matrix_keys)
 
     # Export to CSV
-    export_keys = ["UID"]
+    export_keys = ["UID", "operator"]
     export_keys.extend(show_keys)
     export_keys.append("humanitarian_priorities_raw")
     export_keys.extend(demog_keys)
     export_keys.extend(evaluation_keys)
 
-    with open(csv_output_path, "w") as f:
+    # TODO: Output data before folding.
+
+    # Output to CSV with one message per row
+    with open(csv_by_message_output_path, "w") as f:
+        TracedDataCSVIO.export_traced_data_iterable_to_csv(data, f, headers=export_keys)
+
+    sys.setrecursionlimit(1500)
```

Comment:
please elaborate?

Reply:
I was hoping that this would be so temporary that it wouldn't even need to be in a PR, but other REACH pressures mean I still need it. Added a comment and an issue (https://github.com/AfricasVoices/REACH/issues/10)